pkgs/development/libraries/qt-6/modules/qmake2cmake.nix

```diff
@@ -0,0 +1,68 @@
+/*
+sample use
+nixpkgs/pkgs/tools/misc/qarma-for-qt6/default.nix
+*/
+
+{ python3
+, python3Packages
+, git
+, srcs
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "qmake2cmake";
+  inherit (srcs.qtbase) src version;
+  sourceRoot = "qtbase-everywhere-src-${version}/util/cmake";
+
+  propagatedBuildInputs =
+    (with python3Packages; [ pyparsing portalocker sympy pytest ])
```

Comment:
pytest should never be propagated because it makes it very hard to override it. If it is only required for checkPhase please move it to checkInputs, otherwise it goes to buildInputs.

Reply:
done. `sympy pytest` are test deps